plugins/tokenizers/mecab.c

```diff
@@ -1083,6 +1109,11 @@ mecab_init(grn_ctx *ctx, grn_tokenizer_query *query)
     GRN_PLUGIN_FREE(ctx, tokenizer);
     return NULL;
   }
+  tokenizer->lattice = mecab_model_new_lattice(tokenizer->mecab->mecab_model);
```

Comment:
This should not be created when `grn_mecab_lock_free == false`. It will improve performance on parallel case.